packages/ebayui-core/src/components/ebay-number-input/number-input.stories.ts

```diff
@@ -0,0 +1,207 @@
+        invalid: {
+            type: "boolean",
+            control: { type: "boolean" },
+            description:
+                "indicates a field-level error with red border if true",
+        },
+        "aria-label": {
+            type: "string",
+            control: { type: "Options" },
+            description:
+                "Either this or @label is required. Renders text for screen readers",
+        },
+        label: {
+            description: "Either this or aria-label is required. Renders label inside input if set",
+            control: { type: "text" },
+            table: {
+                category: "@attribute tag",
+                defaultValue: {
+                    summary: "",
+                },
+            },
+        },
+        "on-change": {
+            action: "on-change",
+            description: "Triggered whenever the value of the input changes",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, value }",
+                },
+            },
+        },
+        "on-input-change": {
+            action: "on-input-change",
+            description: "Triggered when the value of the input is changed",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, value }",
+                },
+            },
+        },
+        "on-focus": {
+            action: "on-focus",
+            description: "Triggered on focus",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, value }",
+                },
+            },
+        },
+        "on-blur": {
+            action: "on-blur",
+            description: "Triggered on blur",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, value }",
+                },
+            },
+        },
+        "on-keypress": {
+            action: "on-keypress",
+            description: "Triggered on keypress",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, value }",
+                },
+            },
+        },
+        "on-keyup": {
+            action: "on-keyup",
+            description: "Triggered on keup",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, value }",
+                },
+            },
+        },
+
+        "on-keydown": {
+            action: "on-keydown",
+            description: "Triggered on keydown",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, value }",
+                },
+            },
+        },
+       "onIncrement": {
```

Comment:
should it be `on-increment`?

Reply:
We use camel case here because marko translates and understands both